lighthouse-core/test/chromium-web-tests/README.md

```diff
@@ -14,6 +14,13 @@ yarn test-devtools
 yarn update:test-devtools
 ```
 
+### Debugging
+
+* Want logs from Lighthouse? Add `log.log('status', '**** hello test output ' + JSON.stringify({obj}));` which will be visible in the `lighthouse-successful-run.js` output thanks to `LighthouseTestRunner.addStatusListener`.
+* Want logs from test files? Adding `--driver-logging --no-retry-failures` to the `run_web_tests.py` command in `web-test-server.sh` will print to terminal.
```

Comment:
We pass all args (`$*`) through to this command so this can just be `yarn test-devtools --driver-logging --no-retry-failures` (haven't tested)